packages/cli/src/__tests__/commands/__snapshots__/Init.test.ts.snap

```diff
@@ -13,8 +13,7 @@ Next steps:
 5. Tip: Explore how you can extend the ORM with scalable connection pooling, global caching, and real-time database events. Read: https://pris.ly/cli/beyond-orm
 
 More information in our documentation:
-https://pris.ly/d/getting-started
-    "
```

Comment:
I was battling against this change. In the end decided to update the snapshots. 

The default output looks good:

![image](https://github.com/user-attachments/assets/eaa38422-fe1e-41cc-8818-f3bb5dfec4c9)